src/common/utils/get-duration.util.ts

```diff
@@ -39,6 +39,9 @@ export function getDuration(seconds: number) {
     if (value > 0 || (seconds == 0 && i == 3)) {
       const word = pluralize(value, "", pluralizations[values[i]]);
       return_value += `${value} ${word} `;
```

Comment:
```
return_value += `${value} ${word} `;
```
remove extra space after ${word}. May cause some several minutes to be wasted when debugging why and "from where I have this fucking extra space" in the future

Reply:
> ```
> return_value += `${value} ${word} `;
> ```
> 
> remove extra space after ${word}. May cause some several minutes to be wasted when debugging why and "from where I have this fucking extra space" in the future

Please check original comment

Reply:
![F0zTZ2BXoAEuGUG](https://github.com/shysecre/yamaoka-discord/assets/106016833/02987889-2e02-41f2-beb4-e72f12ab7442)